### DISCUSSION TITLE
How to use singularity run qdrant?

### DISCUSSION THREAD

**Discussion by chihchiehchen** [qdrant/qdrant#4504] [General]
Hello,

Thanks for sharing your great project. I use 'docker run qdrant/qdrant' and it works well. However, I would like to use qdrant on the server, which I don't have root permissions. Therefore I tried to use singularity to exec qdrant but didn't succeed.

Could you give me some hints? In any case thanks for your patient and help!

**Comment by xzfc** [comment]
Hello.
Have you tried to run Qdrant executable directly without using a container?
You could download it here: https://github.com/qdrant/qdrant/releases/tag/v1.9.5

**Comment by chihchiehchen** [comment]
Oh, I see, I never thought about it. Thanks for kindly reminding me.

Finally I did not succeed and got error message 
"Unable to find libclang: "couldn't find any valid shared libraries matching: ['libclang.so', 'libclang-*.so', 'libclang.so.*', 'libclang-*.so.*'], set the `LIBCLANG_PATH` environment variable to a path where one of these files can be found (invalid: [])""

which cannot be resolved without root permission. I will talk to the IT manager in our lab. 

In any case, thanks a lot!
